src/engine-scripts/puppet/sidebarState.js

```diff
@@ -0,0 +1,5 @@
+const menuState = require( './menuState' );
+module.exports = async (page, hashtags) => {
+    const isClosed = hashtags.includes( '#sidebar-closed' );
+    menuState(page, '#mw-sidebar-button', isClosed);
```

Comment:
I think this might need either an `await` or `return`?

Reply:
Doe in follow up.